src/forms/NewLizardEntry.jsx

```diff
@@ -73,8 +75,52 @@ export default function NewLizardEntry() {
       setFenceTraps(fenceTrapsArray)
     }
     getAnswerFormDataFromFirestore()
+    const fetchToeCodes = async () => {
+        let toeCodesSnapshot;
+        try {
+            toeCodesSnapshot = await getDocFromCache(
+                doc(db, 'TestToeClipCodes', currentData.site)
+            );
+            console.log('getting toe codes from test');
+            setSiteToeCodes(toeCodesSnapshot.data());
+        } catch (e) {
+            console.log('getting toe codes from live');
+            toeCodesSnapshot = await getDocsFromCache(
+                query(collection(db, 'ToeClipCodes'), where('SiteCode', '==', currentData.site))
+            );
+            setSiteToeCodes(toeCodesSnapshot.docs[0].data());
+            // console.log('retreiving toe codes from ' + currentData.site)
```

Comment:
Do you need the comments still? Everything's looking good so far.

Reply:
for testing